docs/blog/newsletter-april-2020.rst

```diff
@@ -0,0 +1,98 @@
+.. post:: April 03, 2020
+   :tags: newsletter
+
+######################################
+Write the Docs Newsletter – April 2020
+######################################
+
+Hello, friends. Beth A here, and it feels pretty strange to be sitting down to write the usual newsletter intro to you all, when things in the world and at home are far from usual.
+
+But pandemics aside, there's still a lot happening online in the Write the Docs community, and I hope that this newsletter can bring you something interesting - or at least distracting - in trying times. And if you're in need of some support - whether you're struggling with the current crisis, or with regular documentarian problems - if you need someone to talk to, the community is here for you on `Slack </slack/>`__.
+
+In this month's edition, we've got our usual wide range of topics. As many folks are working remotely for the first time (your editor included - I'm four weeks in), we have some advice from the community on how to manage that. We talk about when keeping docs and code in the same repository might not be the right choice; we look at the `salary survey results <>`__ went down; and from the #learn-tech-writing book club, we have an introduction to the principles of 'Every Page is Page One'.
+
+That's it from me. Thinking of you all.
+
+------------------------------
+Adapting to working from home
+------------------------------
+
+It's no surprise that there's been a lot of discussion about how to work effectively from home. So here's a selection of thoughts that folks on Slack said they'd found helpful:
+
+- While there's plenty of good advice on how to work productively from home, remember that the situation you're in is **not** normal WFH. At best, you're cooped up involuntarily at a time when the news cycle is extremely present; and you may be ill, or caring for your family, or worried to be apart from them. So take any advice with that pinch of salt. Above all, be kind to yourself.
+- We could be in this situation for a while, so it's worth gearing up for that. There was strong agreement that it's worth maintaining a routine - it's easy to drift into spending a day in pyjamas, but that can leave you exhausted. Many find it helpful to have a defined start to the work day by getting up and dressed as usual; some even take 'commute' time to walk around the block or listen to podcasts. 
+- It's just as important to finish the day, rather than drifting into working late: setting boundaries is the name of the game. If it's possible for you, helps to have a defined work space away from other activities. And similar to the morning, an end-of-day 'commute' can help you switch off. 
```

Comment:
```suggestion
- It's just as important to finish the day, rather than drifting into working late: setting boundaries is the name of the game. If it's possible for you, it helps to have a defined work space away from other activities. And similar to the morning, an end-of-day 'commute' can help you switch off. 
```

Reply:
Should probably expand WFH :-p